core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -245,13 +270,17 @@ public List<TableIdentifier> listTables(Namespace namespace) {
         row ->
             JdbcUtil.stringToTableIdentifier(
                 row.getString(JdbcUtil.TABLE_NAMESPACE), row.getString(JdbcUtil.TABLE_NAME)),
-        JdbcUtil.LIST_TABLES_SQL,
+        JdbcUtil.listTablesSql(),
         catalogName,
         JdbcUtil.namespaceToString(namespace));
   }
 
   @Override
   public void renameTable(TableIdentifier from, TableIdentifier to) {
+    if (viewExists(to)) {
+      throw new AlreadyExistsException("Cannot rename %s to %s. View already exists", from, to);
```

Comment:
Nit: I think a shorter exception message would work here. 

```
throw new AlreadyExistsException("View %s already exists", to);
```

Reply:
All the catalog follow the same error message as testcase expects it. 

https://github.com/apache/iceberg/blob/70b7aa534b2c79ccd7b6c0e0fd1be980772bb20a/core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java#L688

Reply:
@amogh-jahagirdar I'm using the expected exception message to have the tests happy 😄